iquip/apps/builder.py

```diff
@@ -459,14 +494,18 @@ def initArgsEntry(self, experimentInfo: ExperimentInfo):
         for argName, (argInfo, *_) in experimentInfo.arginfo.items():
             # TODO(BECATRUE): The other types such as 'Scannable'
             # will be implemented in Basic Runner project.
+            argType = argInfo.pop("ty")
             entryCls = {
                 "BooleanValue": _BooleanEntry,
                 "StringValue": _StringEntry,
                 "EnumerationValue": _EnumerationEntry,
-                "NumberValue": _NumberEntry
-            }[argInfo.pop("ty")]
+                "NumberValue": _NumberEntry,
+                "Scannable": _ScanEntry
+            }[argType]
             widget = entryCls(argName, argInfo)
-            item = QListWidgetItem(self.builderFrame.argsListWidget)
+            listWidget = self.builderFrame.scanListWidget if argType == "Scannable" \
+                         else self.builderFrame.argsListWidget
+            item = QListWidgetItem(listWidget)
             item.setSizeHint(widget.sizeHint())
             self.builderFrame.argsListWidget.addItem(item)
             self.builderFrame.argsListWidget.setItemWidget(item, widget)
```

Comment:
I think you need to replace these two `self.builderFrame.argsListWidget` with `listWidget`.